.github/workflows/docker-publish.yml

```diff
@@ -98,3 +98,46 @@ jobs:
           aws ecr get-login-password --region ${{ secrets.ENV_NOVI_ECR_AWS_REGION }} | \
           docker login --username AWS --password-stdin "${AWS_ECR_ACCOUNT_URL}"
           docker/docker_republish.sh -t ${BRANCH}_${GIT_REV} -r ${AWS_ECR_ACCOUNT_URL} -d
+
+  run-perf-test:
+    runs-on: self-hosted
+    timeout-minutes: 120
+    needs: build-images
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Determine changes
+        uses: diem/actions/changes@266156eaff7ba6e1253cf7019506c7b3702f73de
+        with:
+          workflow-file: docker-publish.yml
+      - name: Run cluster test
+        run: |
+          BRANCH=$(echo "$GITHUB_REF" | sed 's|.*/||' )
+          GIT_REV=$(git rev-parse --short=8 HEAD)
+          IMAGE_TAG="${BRANCH}"_"${GIT_REV}"
+          ./scripts/cti --tag $IMAGE_TAG --timeout-secs 7200 \
```

Comment:
do we need to run it as long as 2 hours? I assume it will occupy 1 of 3 available clusters which may block some of our pipe lines

Reply:
The test suite takes ~1.5 hr.  2hr is to give it enough buffer for waiting for an available cluster.

Reply:
oh shoot, just realized this likely not going to work w/o enforced queueing in GHA. 

It takes ~45min to land a PR, and it takes twice as long to run a full suite on it post-land.  While the full suite is running against the previously landed PR, another PR has landed and will start another full suite (GHA doesn't queue at the moment). This just leaves one remaining cluster for land and canary.  On a busy day, this will lead to  contention.